sdcm/sct_events.py

```diff
@@ -234,6 +234,27 @@ def __str__(self):
                 super(DisruptionEvent, self).__str__(), self)
 
 
+class FullScanEvent(SctEvent):
+    def __init__(self, type, loader_node, ks_cf, db_node_ip, result=None):
+        super(FullScanEvent, self).__init__()
+        self.type = type
+        self.loader_node = str(loader_node)
+        self.ks_cf = ks_cf
+        self.db_node_ip = db_node_ip
+        self.result = result
+        self.publish()
+
+    def __str__(self):
+        msg = "{0}: type={1.type} loader_node={1.loader_node} select_from={1.ks_cf} on db_node={1.db_node_ip}"
+        if self.result:
+            if self.result['stderr']:
+                self.severity = Severity.ERROR
+            msg += " {1.result}"
```

Comment:
I don't know what's in results, but if it's lots of data, please put it into a new line.

i.e. if you can paste here an example of the output from `events.log` that would be nice. we also want that to be clear and readable

Reply:
@fruch  the output is next:
2019-05-22 09:37:51.518: (FullScanEvent Severity.NORMAL): type=finish loader_node=Node abykov-test-abykov-loader-node-2a3e79a0-1 [52.90.45.100 | 172.30.0.62] (seed: False) select_from=keyspace1.standard1 on db_node=172.30.0.36 {'stderr': '', 'stdout': '\n count\n-------\n  3000\n\n(1 rows)\n'}

Reply:
it should be too large.

Reply:
longer the scylla decoded backtraces ? if it's 20-30 lines that should be fine. if it can be more then that, it's far better then having all this in one liner that isn't readable.

in cases of error we want to see any info that can help us. and we want it clear.